### ISSUE TITLE
Next event

### ISSUE THREAD

**Issue by eirektg** [ottopaulsen/node-red-contrib-power-saver#138]
Would it be nice to have the output from node 3 telling about when the next event is going to happen? In other words when it's going to switch from on to off, or the opposite. It would be nice to easily be able to present the time & date in a dashboard, or other logic.

I thinking about the same thing that shows as an attribute in the quite new Schedule integration (helper) in home assistant:
![image](https://user-images.githubusercontent.com/48910831/207590464-d4e6ac66-9565-481f-b837-cc64d3907364.png)

**Comment by ottopaulsen** [comment]
I believe it should be possible to create a function node reading from output 3 and creating what you want.

**Comment by eirektg** [comment]
Yes, i guess so, but I was hoping that this value could be available as part of the output of node 3. In that case it's way easier to actually get this number, and there will also fewer nodes in Node Red to make this logic work

Not important at all :-)
